### ISSUE TITLE
Problem with cache and encryption

### ISSUE THREAD

**Issue by pekapa** [bradleyg/django-s3direct#107]
Hello,

I'm trying to set the cache control as specified in the docs, with 
```
        'cache_control': 'max-age=2592000', # Default no cache-control 
```
but amazon always return my file with a `pragma: no-cache` what am I doing wrong?

Also, I think I'm missing some step with the encryption... whenever I set
```
        'server_side_encryption': 'AES256', # Default no encryption
```
I get a forbidden from amazon side, what do I need to change to enable it?

**Comment by elioscordo** [comment]
Hello, 
even with no cache the admin widget fails with 

`'server_side_encryption': 'AES256', # Default no encryption`

Any ideas / workaround?

**Comment by bradleyg** [comment]
Pretty sure this was fixed in #114
